README.md

```diff
@@ -22,6 +22,9 @@ Connect your device or start the emulator and run the following code
 # change directories
 cd Yoga-Guru
 

```

Comment:
Can you remove the changes below!
These have already been merged in the previous PR #9